packages/server/dataloader/atlassianLoaders.ts

```diff
@@ -230,8 +230,9 @@ export const jiraIssue = (
             const possibleEstimationFieldNames = [] as string[]
             Object.entries<{schema: {type: string}}>(issueRes.editmeta?.fields)?.forEach(
               ([fieldId, {schema}]) => {
-                if (isValidEstimationField(schema.type, issueRes.names[fieldId], fieldId)) {
-                  possibleEstimationFieldNames.push(issueRes.names[fieldId])
+                const fieldName = issueRes.names[fieldId]
+                if (fieldName && isValidEstimationField(schema.type, fieldName, fieldId)) {
+                  possibleEstimationFieldNames.push(fieldName)
                 }
                 if (schema.type === 'timetracking') {
```

Comment:
could this add duplicate fields?

Reply:
Timetracking fields were not listed separately in the editmeta.